agent/apps/ai_helsabrain/app_code/cores/speech_processor.py

```
@@ -0,0 +1,8 @@
+"""
+هسته پردازش صوت برای اپلیکیشن
+Speech Processing Core for Application
+"""
+
+from app_standards.four_cores import SpeechProcessorCore
```

Ruff (0.12.2)

`app_standards.four_cores.SpeechProcessorCore` imported but unused

Remove unused import: `app_standards.four_cores.SpeechProcessorCore`

(F401)